src/components/People/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,98 @@
+import { FC } from 'react';
+import classNames from 'classnames';
+
+import { Person } from '../../../types';
+import { findParent } from '../../../helpers';
+import { PeopleLink } from '../PeopleLink';
+import { SortType } from '../../../enums/SortType';
+import { SortLink } from '../../SortLink';
+
+type Props = {
+  people: Person[],
+  personId: string,
+};
+
+export const PeopleTable: FC<Props> = (props) => {
+  const {
+    people,
+    personId,
+  } = props;
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          {[...Object.keys(SortType)]
+            .filter((type) => type.toLowerCase() !== SortType.NONE)
+            .map((type) => (
+              <th>
+                <span className="is-flex is-flex-wrap-nowrap">
+                  {type.slice(0, 1) + type.slice(1).toLowerCase()}
```

Comment:
Another good way to do such a transformation it's using a CSS rule `text-transform: capitalize;`. But this CSS will make only the first letter capitalized, so it will not work if the text is uppercased, but in your example, it'll be fine.